content/development/how-to-write-a-loader.md

```diff
@@ -9,7 +9,7 @@ This function is called when a resource should be transformed by this loader.
 
 In the simple case, when only a single loader is applied to the resource, the loader is called with one parameter: the content of the resource file as string.
 
-The loader can access the [[loader API | loaders]] on the `this` context in the function.
+The loader can access the [[loader API | loaders]](api/loaders/) on the `this` context in the function.
```

Comment:
I thought so but I'd seen this double brackets elsewhere and thought it was specific to something else. Thanks for the clarification! Will update it now 👍